assets/queries/openAPI/2.0/basepath_with_wrong_format/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	not regex.match("^/", doc.basePath)
```

Comment:
shouldn't this be:

```suggestion
	not regex.match("^/.*", doc.basePath)
```

/anything/else ??

Reply:
No, currently regex is correct, as you can see in [specification json's schema](http://23.22.16.221/v2/schema.json)

Reply:
As Felipe said above I followed the specification JSON's schema